src/index.js

```diff
@@ -261,33 +265,35 @@ const InputMask = forwardRef(function InputMask(props, forwardedRef) {
     setInputState(newInputState);
   });
 
+  const refCallback = node => {
+    inputRef.current = node;
```

Comment:
First main change is here - to remove `findDOMNode` - however that alone breaks the `children` usage.